brave/brave-core
# move RewardsInternalsInfo to mojo type and use callback

Resolves brave/brave-browser#5131.
Resolves brave/brave-browser#5132.
Resolves brave/brave-browser#5146.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Enable rewards
2. Navigate to `brave://rewards-internals`
3. Notice browser does not crash and shows correct info.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
